### ISSUE TITLE
user module should be able to configure RBAC attributes on Solarish systems

### ISSUE THREAD

**Issue by xen0l** [ansible/ansible#19179]
_From @xen0l on June 12, 2016 17:12_

##### ISSUE TYPE
- Feature Idea
##### COMPONENT NAME

user
##### ANSIBLE VERSION

```
2.1.0.0
```
##### CONFIGURATION

Default
##### OS / ENVIRONMENT

OpenIndiana/SmartOS/Oracle Solaris
##### SUMMARY

illumos/Solaris RBAC provides authorizations/profiles/roles attributes. These should be configurable via Ansible
##### STEPS TO REPRODUCE

```
- name: Configure RBAC profiles
  user: name=xenol
           profile='Primary Administrator'

- name: Configure RBAC authorization
  user: name=xenol
           authorization='zfssnap'

- name: Configure RBAC role
  user: name=xenol
           role='root'
```
##### EXPECTED RESULTS

RBAC authorization/profile/role will be assigned to the user account.

_Copied from original issue: ansible/ansible-modules-core#3936_

**Comment by xen0l** [comment]
_From @alikins on June 13, 2016 15:16_

ping @grossws

**Comment by xen0l** [comment]
_From @grossws on June 15, 2016 0:0_

I have no Solaris/Illumos systems, so it out of scope of my interests currently.

Feel free to make a pull request if you want to contribute a patch.

**Comment by xen0l** [comment]
_From @grossws on July 29, 2016 18:18_

@ansibot, needs_contributor

**Comment by xen0l** [comment]
_From @ansibot on October 6, 2016 14:33_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by xen0l** [comment]
_From @ansibot on December 9, 2016 19:52_

This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
cc @sfromm
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @samdoran
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
